crates/aptos/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 
 ## Unreleased
 
+## [4.3.1] - 2024/10/31
```

Comment:
@fEst1ck, do we plan to release 4.3.1 for this typo? If not, you could add it under `Unreleased`.

Reply:
No. Thanks for pointing out